README.md

```diff
@@ -1,50 +1,163 @@
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+  
+  ## Built With
+  <blockquote>
+
+  **Client:** 
+
+  <a href="">[![React](https://img.shields.io/badge/React-20232A?style=for-the-badge&logo=react&logoColor=61DAFB)](https://reactjs.org])</a>
+
+  <a href="">![CSS Modules](https://img.shields.io/badge/CSS%20Modules-000000?style=for-the-badge&logo=CSS-Module&logoColor=white)</a>
+</blockquote>
+<blockquote>
+
+**Server:** 
+
+<a href="">[![Express](https://img.shields.io/badge/Express%20Js-yellow?style=for-the-badge&logo=expressJS)](https://expressjs.com/)</a>
+
+
+<a href="">[![Mongo](https://img.shields.io/badge/MongoDB-success?style=for-the-badge&logo=mongodb&logoColor=white&style=flat)](https://www.mongodb.com/)
+
+</blockquote>
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+## Getting Started
+
+  > To set up the project locally, follow these simple steps below:
+
+### Prerequisites:
+
+  > Should have **Node.js** installed to get yarn package.
+
+### Run project locally
+
+1. Fork and Clone the repository
+
+```bash
+  git clone https://github.com/webdev-20/shorti-url-shortener.git
+```
+
+2. Go to the project directory
+
+```bash
+  cd shorti-url-shortener
+```
+
+3. Install dependencies in both client and server
+
+```bash
+  yarn install
+```
+
+4. Start the server
+
+```bash
+  yarn start (for both client and server)
+  yarn client (for client only)
+  yarn server (for server only)
+```
+ **Note:** Create and fill in server/.env and client/.env 
+
+ More details [here](https://github.com/webdev-20/shorti-url-shortener/blob/main/CONTRIBUTE.md)
+
+ <p align="right">(<a href="#top">back to top</a>)</p>
+ 
+## Contributing
+
+We welcome your contribution!
+
+Please get the issue assigned to yourself before working on it, to avoid duplicate efforts.
+
+Any Suggestions, please fork the repo and create a pull request.
+Or create an issue with tag - **enhancement**
+
+1. Fork the repository
+2. Clone it and navigate to your local repository
+3. Create your feature branch, example:- fix/header
+4. Stage changes & commit
+5. Create a pull request
+
+See [contributing.md](https://github.com/webdev-20/shorti-url-shortener/blob/main/CONTRIBUTE.md) for ways to get started and more details.
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## API Documentation
+
+Details of API endpoints can be viewed here
+https://localhost:4002/api/docs
+
+## Features
+
+- [ ] Track frequency of shortened URL clicked
+- [ ] Track Referrer
+- [ ] Create groups of shortened links for campaings or themes 
+  <br>
+
+Visit the [open issues](https://github.com/webdev-20/shorti-url-shortener/issues) for a full list of proposed features. 
+
+## Live Site
+
+Visit our production site here - 
 https://shorti.onrender.com/
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## License
+
+[MIT](https://choosealicense.com/licenses/mit/)
+
+## Contacts
+
+[Cherylli](https://github.com/cherylli) <br> **Discord ID**- Notcori#0144
```

Comment:
Can you change this to [Cheryl M](https://github.com/cherylli) please. Thank you!

Reply:
sure... this is taken care of.